relay-server/src/actors/store.rs

```diff
@@ -199,51 +199,122 @@ impl StoreService {
             }
         }
 
+        let remote_addr = envelope.meta().client_addr().map(|addr| addr.to_string());
+
+        let kafkamessages = self.extract_kafka_messages(
+            event_item,
+            event_id,
+            scoping,
+            start_time,
+            remote_addr,
+            &mut attachments,
```

Comment:
The signature of the function can be changed and then just move the attachments there.
```suggestion
           attachments,
```